frontend/packages/ceph-storage-plugin/console-extensions.json

```
@@ -1 +1,154 @@
-[]
+[
+  {
+    "type": "console.storage-class/provisioner",
+    "properties": {
+      "csi": {
+        "openshift-storage.rbd.csi.ceph.com": {
+          "title": "Ceph RBD",
+          "provisioner": "rbd.csi.ceph.com",
+          "allowVolumeExpansion": true,
+          "parameters": {
+            "clusterID": {
+              "name": "Cluster ID",
+              "hintText": "The namespace where Ceph is deployed",
```

Localize all strings in this file:
```suggestion
              "name": "%namespace~Cluster ID%",
              "hintText": "%namespace~The namespace where Ceph is deployed%",
```

---

@christianvogt the default provisioners are also not translated. Do we need to translate these?

---

All strings in console should be translated. But seems like that should be a separate PR.